sdk/servicebus/Microsoft.Azure.ServiceBus/src/Amqp/AmqpExceptionHelper.cs

```diff
@@ -263,5 +263,18 @@ public static Exception GetInnerException(this AmqpObject amqpObject)
 
             return innerException == null ? null : GetClientException(innerException, null, null, connectionError);
         }
+
+        public static bool TryTranslateToRetriableException(Exception exception, out ServiceBusException retriableException)
```

Comment:
I realize we don't have tests for this class and not sure if I should add a test for this method. I took this class from Serkant's PR fix.